sql/load_sql_context.sql

```diff
@@ -359,7 +360,8 @@ select
                                                 select
                                                     jsonb_build_object(
                                                         'name', d.directive ->> 'name',
-                                                        'description', d.directive -> 'description'
+                                                        'description', d.directive -> 'description',
+                                                        'mappings', (graphql.comment_directive(pg_catalog.obj_description(pa.atttypid, 'pg_type'))) -> 'mappings'
```

Comment:
Let's look up mappings for columns as needed by traversing the SQL context in rust

I'd like to keep the SQL context free of all duplication

Reply:
See the comment below